spec/pom.xml

```diff
@@ -20,7 +20,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>jakarta.ejb</groupId>
     <artifactId>jakarta.ejb-spec</artifactId>
-    <version>4.0-SNAPSHOT</version>
+    <version>4.0</version>
```

Comment:
I thought the goal of the changes bellow were to remove the link between the spec release and the artifact release. 
So why changing this one to 4.0?

Reply:
The revnumber attribute used in adoc is derived from project version. When generating the PDF or HTML it is currently displaying the version as 4.0-SNAPSHOT. Either we can change the revnumber attribute to point to a different property or drop the -SNAPSHOT identifier from version.

What do you suggest?

Reply:
Locally, I have updated the rev in the docs to use the spec.version that I added in the spec project because it's already in the api and parent pom.

Are you fine with that?
You want to update the PR?

Reply:
Other than that, the file naming is fine with me

Reply:
I have updated the revnumber to use spec.version property instead of project.version.